TF_Settings_Web/src/Components/ScreenManager.tsx

```diff
@@ -0,0 +1,112 @@
+// Provides an "ActiveScreen" / "ActiveTab" component that then contains the correct screen
+// Manages the loading/display of the subscreens
+// Is the place where the active screen/tab is controlled
+
+import React, { Component, CSSProperties } from "react";
+import { ControlBar } from "./ControlBar";
+import { CursorManager } from "./CursorManager";
+import { CameraPage } from "./Pages/CameraPage";
+import { InteractionsPage } from "./Pages/InteractionsPage";
+
+const TouchFree = window.TouchFree;
+
+interface ScreenManagerState {
+    atTopLevel: boolean,
+    tfState: number,
+    activeTabName: string,
+}
+
+const pages: {[name: string]: typeof Component} = {
+    "Camera": CameraPage,
+    "Interactions": InteractionsPage,
+};
+
+export class ScreenManager extends React.Component<{}, ScreenManagerState> {
+    private containerStyle : CSSProperties = {
+        display: 'flex',
+        flexDirection: 'column',
+        height: '100%'
+    }
+
+    private timerID: number;
+
+    private cursorManager: CursorManager;
+    private addCursorsToDoc: (element: HTMLDivElement) => void;
+
+    // TouchFree objects
+    private inputSystem: any;
+    private connectionManager: any;
+
+    constructor(props: {}) {
+        super(props);
+
+        this.timerID = -1;
+
+        TouchFree.Connection.ConnectionManager.init();
+        this.inputSystem = new TouchFree.InputControllers.WebInputController();
```

Comment:
Can we import the classes from tooling to make them a bit easier to use? e.g. `import { WebInputController } from "../TouchFree/TouchFree_Tooling"`